src/components/AnimatePresence.tsx

```diff
@@ -211,8 +175,17 @@ export const AnimatePresence: FunctionComponent<AnimatePresenceProps> = ({
     custom,
     initial = true,
     onExitComplete,
-    _syncLayout,
 }) => {
+    // Use a running state counter to allow us to force a re-render once all
+    // exiting animations have settled. For performance reasons it might also be
+    // necessary to track entering animations too, but this should be good enough
+    // for the majority use-cases ie slideshows, modals etc.
+    const [forcedRenderCount, setForcedRenderCount] = useState(0)
+    const incrementForcedRenderCount = () =>
+        setForcedRenderCount(forcedRenderCount + 1)
```

Comment:
Shouldn't this be wrapped in a useCalback?
This also seems like it is duplicated from `SharedLayout` so it can be extracted in a `useForceUpdate` hook